src/middlewared/middlewared/etc_files/rancher/k3s/flags.py

```diff
@@ -34,7 +34,14 @@ def render(service, middleware):
     kubelet_args = [
         'max-pods=250',
     ]
+
+    disabled_features = []
+    if not config['servicelb']:
+        disabled_features.append("servicelb")
```

Comment:
Can we please be consistent about quotes usage and use the standard being followed throughout the file i.e single quotes ?

Reply:
agreed.
```suggestion
        disabled_features.append('servicelb')
```